### DISCUSSION TITLE
Render the "Entry Type" element index attribute with icons and colors

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#15922] [Ideas] [answered]
It'd be great if entry type element index attributes included the entry types' icons and colours (i.e. similarly to how they appear in entry type selectors in element edit pages), instead of just rendering the type names as plain text. 

![CleanShot 2024-10-18 at 15 48 00](https://github.com/user-attachments/assets/be9d11d5-e266-4ecf-a1a8-3416d42bd370)

![CleanShot 2024-10-21 at 01 09 50](https://github.com/user-attachments/assets/ea3facd8-a4c1-4e70-b421-cf4a1d1db131)

**Comment by piotrpog** [comment]
Very good idea.

**Comment by brandonkelly** [comment, marked as answer]
Good idea! Just made that change for Craft 5.5 (db94742c40b6ec8896dd552b2ab30f56a7d02ef4).

![An entry index, showing how the two “Article” and “Link” entry types are now displayed with their icons and colors](https://github.com/user-attachments/assets/8acb9804-c863-4c09-9867-53511bfcdd35)

**Reply by brandonkelly** [reply]
Craft 5.5.0 is out now with that change 🎉
